spec/lib/opentoken-lib.spec.js

```diff
@@ -0,0 +1,43 @@
+"use strict";
+
+describe("opentokenLib", () => {
+    var opentokenLib, req, signed;
+
+    // Just some test data that is constant.
+    const account = {
+            code: "code",
+            secret: "secret",
+            id: "accountId"
+        },
```

Comment:
I am unsure how we proceed with `const`.  I think I like your formatting and I suggest you add it to the style guide.